oshi-core/src/main/java/oshi/software/os/unix/aix/AixOperatingSystem.java

```diff
@@ -99,18 +99,16 @@ public InternetProtocolStats getInternetProtocolStats() {
 
     @Override
     public List<OSProcess> getProcesses(int limit, ProcessSort sort) {
-        // Needs updating for flags on AIX or to use /proc fs
         List<OSProcess> procs = getProcessListFromPS(
-                "ps -eo s,pid,ppid,user,uid,group,gid,nlwp,pri,vsz,rss,etime,time,comm,args", -1);
+                "ps -e -o st,pid,ppid,user,uid,group,gid,thcount,pri,vsize,rssize,etime,time,args,comm,pagein", -1);
```

Comment:
`args` needs to be the last thing on the list here.  This is the case where you have to bound the `split()` by the exact number so the `args` (which may contain spaces) doesn't get split up.

Reply:
hmm makes sense.